davinci-resolve: 20.0.1 -> 20.2.1

amarshall opened the pull request:
The "Apple Immersive/Calibration" dir is, like others here, attempted to be created in the store dir during launch and the application exits if it cannot create it; so create it during build.

Changelogs:
https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=225417 https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=226348 https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=227008


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

Supersedes https://github.com/NixOS/nixpkgs/pull/432608

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

amarshall committed:
davinci-resolve: 20.0.1 -> 20.1.1

The "Apple Immersive/Calibration" dir is, like others here, attempted to
be created in the store dir during launch and the application exits if
it cannot create it; so create it during build.

Changelogs:
https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=225417
https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=226348

khaneliman commented:
```bash
error: hash mismatch in fixed-output derivation '/nix/store/pvx7brpryxmzf6lnxls7zvsaq8mrdqra-davinci-resolve-src.zip.drv':
┃          specified: sha256-3vTXDVxYnuDdNJRYAWSnc+zt+52saeY/xXS3j2rGLrI=
┃             got:    sha256-ewym/KX63fnXMg7Czl5/6QI//SXWEyQl0Sd0AzRHzdQ=

khaneliman commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 439617`
Commit: `10a959219e01ebfd279e0cbd24465721e7c3bdf4`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>davinci-resolve</li>
    <li>davinci-resolve-studio</li>
  </ul>
</details>

khaneliman commented:
> @khaneliman I assume you had a network issue when downloading? I have re-validated both srcs have the correct hash.

Yeah, looks like it must have been a transient issue with the cdn.

khaneliman reviewed (approved):
launches fine here

amarshall committed:
davinci-resolve: 20.1.1 -> 20.2.1

Changelog:
https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=227008
https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=227789

khaneliman commented:
Sorry, forgot about this..
